server/src/storage/merchants.ts

```diff
@@ -17,36 +17,14 @@
 /**
  * @fileoverview This file contains the database operations related to Merchant
  * entities.
- * @author Karen Frilya Celine
  */
 
-import {Datastore} from '@google-cloud/datastore';
-import {Guid} from 'guid-typescript';
-
 import {MERCHANT_KIND} from '../constants/kinds';
+import {MerchantPayload} from '../interfaces';
+import {add} from './datastore';
 
-const datastore = new Datastore();
-
-async function getAllMerchants() {
-  const query = datastore.createQuery(MERCHANT_KIND);
-  const [merchants, info] = await datastore.runQuery(query);
-  return merchants;
-}
-
-async function getMerchant(merchantId: string) {
-  const key = datastore.key([MERCHANT_KIND, merchantId]);
-  return await datastore.get(key);
-}
-
-async function addMerchant(merchant: object) {
-  return await datastore.insert({
-    key: datastore.key([MERCHANT_KIND, Guid.raw()]),
-    data: merchant,
-  });
+async function addMerchant(merchant: MerchantPayload) {
+  await add(MERCHANT_KIND, merchant);
```

Comment:
Hmm do we need any response from this?

Reply:
I don't think so, you just need to `await`. If there is any error it would throw an error, which will eventually be caught at the api/merchants.ts. Otherwise, then it is successfully added.

Reply:
Okay based on discussion on Chat, added a response for this to return the inserted id.